src/gcp/storage.js

```diff
@@ -91,9 +91,31 @@ function _deleteObject(location) {
   });
 }
 
+/**
+ * Gets a storage bucket from GCP.
+ * Ref: https://cloud.google.com/storage/docs/json_api/v1/buckets/get
+ * @param {string} bucketName name of the storage bucket
+ * @returns a bucket resource object
+ */
+async function _getStorageBucket(bucketName) {
```

Comment:
Nit: This method is probably just called `getBucket` in the actual API, so we should probably use the same.

Reply:
Changed to getBucket